apps/web/src/locales/en/messages.json

```
@@ -1295,6 +1295,20 @@
   "importEncKeyError": {
     "message": "Error decrypting the exported file. Your encryption key does not match the encryption key used export the data."
   },
+  "importDestination": {
+    "message": "Import destination"
+  },
+  "importTargetHint": {
+    "message": "File structure will import into selected",
+    "description": "Located as a hint under the import target. Will be appended by either folder or collection, depending if the user is importing into an individual or an organizational vault."
+  },
+  "importTargetOrgHint": {
+    "message": "Required, if file contains unassigned items.",
```

The comma is unnecessary and also doesn't match the Figma.

```suggestion
    "message": "Required if file contains unassigned items.",
```